src/Tizen.NUI/src/public/BaseComponents/TextEditorEvent.cs

```diff
@@ -214,8 +214,7 @@ public partial class TextEditor
         /// <summary>
         /// The SelectionStarted event is emitted when the selection start.
```

Comment:
"selection start" -> "selection starts" OR "selection has been started" OR "user starts selecting text"
same for TextField